### ISSUE TITLE
Blank window

### ISSUE THREAD

**Issue by masterwindu69** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1778]
Hey there

OS: Windows 10 Anniversary

I'm still having a blank window after opening the program (v4.0.0). 
![17-10-_2016_04-51-41](https://cloud.githubusercontent.com/assets/22877918/19423990/eafb6d1e-9425-11e6-94f0-45b9dee7efe0.png)
I'm not sure, if related, but under "Audio Devices" it says "Loading Devices...".
![17-10-_2016_04-52-22](https://cloud.githubusercontent.com/assets/22877918/19423989/eafae90c-9425-11e6-928d-376f7afe7f48.png)

Tried to clear cache. Problem persists.

Error can be reproduced by opening the program (in my case on three different Windows 10 machines + one VM).

**Comment by MarshallOfSound** [comment]
I can't replicate this issue. Can you run these commands from Command Prompt and tell me the output.

```bash
set ELECTRON_ENABLE_LOGGING=true
set ELECTRON_ENABLE_STACK_DUMPING=true
%LOCALAPPDATA%\GPMDP_3\app-4.0.0\Google%20Play%20Music%20Desktop%20Player.exe --dev
```

**Comment by christianbearslyclark** [comment]
It gave me all this ... whatever it means LOL

C:\Users\chris>%LOCALAPPDATA%\GPMDP_3\app-4.0.0\Google%20Play%20Music%20Desktop%20Player.exe --dev
'C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\Google%20Play%20Music%20Desktop%20Player.exe' is not recognized as an internal or external command,
operable program or batch file.

**Comment by MarshallOfSound** [comment]
@christianbearslyclark Oh, you need `4.0.0` installed and if it is a fresh install the commands need to be

```bash
set ELECTRON_ENABLE_LOGGING=true
set ELECTRON_ENABLE_STACK_DUMPING=true
%LOCALAPPDATA%\GPMDP_3\app-4.0.0\Google\ Play\ Music\ Desktop\ Player.exe --dev
```

**Comment by christianbearslyclark** [comment]
I have 4.0.0 installed.... same black screen.

I have run the command

**Comment by MarshallOfSound** [comment]
@christianbearslyclark What is the output from that command?  There should be bunch of stuff in your command prompt window?

**Comment by christianbearslyclark** [comment]
oh..... crazy stuff happened LOL


Microsoft Windows [Version 10.0.14393]
(c) 2016 Microsoft Corporation. All rights reserved.

C:\Users\chris>set ELECTRON_ENABLE_LOGGING=true

C:\Users\chris>set ELECTRON_ENABLE_STACK_DUMPING=true

C:\Users\chris>"%LOCALAPPDATA%\GPMDP_3\app-4.0.0\Google Play Music Desktop Player.exe" --dev

C:\Users\chris>
info: Application started.
Unexpected result while probing for avahi: { Error: dns service error: unknown error code
    at Error (native)
    at supportsInterfaceIndexLocalOnly (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\node_modules\mdns\lib\avahi.js:17:12)
    at Object.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\node_modules\mdns\lib\avahi.js:32:20)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\node_modules\mdns\lib\mdns.js:31:19)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\build\main\features\core\websocketAPI.js:9:580)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\build\main\features\core\index.js:9:235)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\build\main\features\index.js:9:14)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at App.<anonymous> (C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\build\index.js:9:3538)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:191:7) errorCode: -65563 }
error: Failed to load bonjour with error: {}
Bonjour is required to use Chromecast Support or to enable ZeroConf for the PlaybackAPI
On windows you need to install Bonjour Print Services
[15144:1016/223041:INFO:CONSOLE(75)] "Download the React DevTools and use an HTTP server (instead of a file: URL) for a better development experience: https://fb.me/react-devtools", source: C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\app.asar\node_modules\react\lib\ReactDOM.js (75)
[15144:1016/223041:INFO:CONSOLE(294)] "::shadow pseudo-element is deprecated. See https://www.chromestatus.com/features/6750456638341120 for more details.", source: C:\Users\chris\AppData\Local\GPMDP_3\app-4.0.0\resources\electron.asar\renderer\web-view\web-view.js (294)
debug: Renderer process logger initialized.
[6728:1016/223044:WARNING:webmediaplayer_impl.cc(341)] Using MultibufferDataSource
[6728:1016/223045:ERROR:ffmpeg_demuxer.cc(1510)] media::FFmpegDemuxer::OnReadFrameDone result=-541478725 IsMaxMemoryUsageReached=0
debug: Renderer process logger initialized.
[4684:1016/223047:WARNING:webmediaplayer_impl.cc(341)] Using MultibufferDataSource
[4684:1016/223047:ERROR:ffmpeg_demuxer.cc(1510)] media::FFmpegDemuxer::OnReadFrameDone result=-541478725 IsMaxMemoryUsageReached=0
[4684:1016/223053:FATAL:memory_win.cc(38)] Out of memory, size = 134217684
Backtrace:
        (No symbol) [0x00D16157]
        (No symbol) [0x00D10C91]
        icu_56::Transliterator::setID [0x032324B1+980161]
        icu_56::Transliterator::setID [0x031CF687+575127]
        icu_56::Transliterator::setID [0x031CBB97+560039]
        (No symbol) [0x00D44CE6]
        (No symbol) [0x00D444B5]
        (No symbol) [0x00D43F5A]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AE00+1132752]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AB50+1132064]
        (No symbol) [0x00BDD6DB]
        (No symbol) [0x00BD5A5A]
        (No symbol) [0x00BD349C]
        (No symbol) [0x00BD395D]
        (No symbol) [0x00BD3908]
        (No symbol) [0x00BD37F5]
        (No symbol) [0x00BD3797]
        v8::Locker::IsLocked [0x555477B3+1229843]
        v8::Locker::IsLocked [0x55434E4A+105130]
        v8::Locker::IsLocked [0x5543DF47+142247]

Backtrace:
        GetHandleVerifier [0x00D620E9+115689]
        icu_56::Transliterator::setID [0x032324B1+980161]
        icu_56::Transliterator::setID [0x031CF687+575127]
        icu_56::Transliterator::setID [0x031CBB97+560039]
        (No symbol) [0x00D44CE6]
        (No symbol) [0x00D444B5]
        (No symbol) [0x00D43F5A]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AE00+1132752]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AF50+1133088]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AFB0+1133184]
        icu_56::RuleBasedBreakIterator::operator!= [0x0176AB50+1132064]
        (No symbol) [0x00BDD6DB]
        (No symbol) [0x00BD5A5A]
        (No symbol) [0x00BD349C]
        (No symbol) [0x00BD395D]
        (No symbol) [0x00BD3908]
        (No symbol) [0x00BD37F5]
        (No symbol) [0x00BD3797]
        v8::Locker::IsLocked [0x555477B3+1229843]
        v8::Locker::IsLocked [0x55434E4A+105130]
        v8::Locker::IsLocked [0x5543DF47+142247]
        (No symbol) [0x3E20A0DE]
        (No symbol) [0x2E97F153]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x2A3A3A59]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x33B2210C]
        (No symbol) [0x1242B7D3]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x33B4B21F]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x1B3D34DD]
        (No symbol) [0x1B3D2E93]
        (No symbol) [0x33BBBC4F]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x33BCDEDE]
        (No symbol) [0x33BCDC5A]
        (No symbol) [0x33BCDA35]
        (No symbol) [0x33B0E6FF]
        (No symbol) [0x33B0E2BC]
        (No symbol) [0x33B0E074]
        (No symbol) [0x1247BEA1]
        (No symbol) [0x33B0E6FF]
        (No symbol) [0x33B0E2BC]
        (No symbol) [0x33B0E074]
        (No symbol) [0x3E20B5D6]
        (No symbol) [0x33BCD40A]
        (No symbol) [0x1B3D34DD]
        (No symbol) [0x1B3D2E93]
        (No symbol) [0x124DA4F3]
        (No symbol) [0x3E2394DE]
        (No symbol) [0x3E2245C3]
        v8::String::WriteUtf8 [0x55609693+114067]
        v8::String::WriteUtf8 [0x55608EE5+112101]
        v8::Function::Call [0x555D02D4+468]
        icu_56::RegexPattern::operator!= [0x02A2CBCE+3166542]
![capture](https://cloud.githubusercontent.com/assets/11014645/19426322/654145d4-93f0-11e6-965d-7dcd78af4330.PNG)

**Comment by MarshallOfSound** [comment]
@christianbearslyclark Right, that's a log I can work with. 

Can you jump on Gitter real quick so we can try hash out a fix 👍

**Comment by MarshallOfSound** [comment]
Fixed in Electron `1.4.3` 

Confirmed by @christianbearslyclark

**Comment by itroy187** [comment]
yay! Thanks a lot! It works!
